.gitignore

```diff
@@ -22,3 +22,9 @@ typechain/
 test-results.json
 scenario-results.json
 
+#certora
+.certora*
+.certora*.json
+**.last_conf*
+certora-logs
+resource_errors.json
```

Comment:
Can this one be deleted with the `.gitignore` below, or these are created at the top-level?